lib/gluegun.rb

```diff
@@ -32,8 +44,8 @@ def self.gluegun_generate_pages(site_config_file)
       FileUtils.mkdir_p(dest_path) unless File.exist?(dest_path)
       YAML.load(open(site_config_file).read).each do |key, value|
         value.each do |key2, value2|
-          File.open(File.join(dest_path,"/#{key2["slug"]}.html"), "w+") do |f|
-            f.puts(GitHub::Markdown.render_gfm(open(key2["link"]).read))
+          File.open(File.join(dest_path,"/#{key2['slug']}.html"), "w+") do |f|
+            f.puts(GitHub::Markdown.render_gfm(open(raw(key2['link'])).read))
```

Comment:
This may be too deep to use here. Let's do a validate at the top level and use render_gfm on only valid urls/files.